compiler/src/dotty/tools/dotc/transform/ElimRepeated.scala

```diff
@@ -47,6 +48,8 @@ class ElimRepeated extends MiniPhase with InfoTransformer { thisPhase =>
 
   private def overridesJava(sym: Symbol)(implicit ctx: Context) = sym.allOverriddenSymbols.exists(_.is(JavaDefined))
 
+  private def hasVargsAnnotation(sym: Symbol)(using ctx: Context) = sym.hasAnnotation(requiredClass[scala.annotation.varargs])
```

Comment:
The usual pattern for referring to a class in the standard library is to add it to Definitions.scala, probably next to the other annotations: https://github.com/lampepfl/dotty/blob/baba3f0a610bf971045d69a29d7f9e9c1ce2c8ff/compiler/src/dotty/tools/dotc/core/Definitions.scala#L791

Reply:
I then used CtxLazy like [CheckReentrant](https://github.com/lampepfl/dotty/blob/master/compiler/src/dotty/tools/dotc/transform/CheckReentrant.scala) but this is much better indeed! done